Assets/HoloToolkit/Input/Tests/Scripts/FocusedObjectMessageReceiver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// FocusedObjectMessageReceiver class shows how to handle messages sent by FocusedObjectMessageSender.
+/// This particular implementatoin controls object appearance by changing its color when focused.
+/// </summary>
+public class FocusedObjectMessageReceiver : MonoBehaviour
```

Comment:
Technically I can achieve the same functionality with just GazeManager.cs.
What are we providing extra with this feature?